src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ReviewChangeStartYearRequest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sfa.Tl.ResultsAndCertification.Models.Contracts.AdminDashboard
+{
+    public class ReviewChangeStartYearRequest
+    {
+        public int PathwayId { get; set; }        
+        public long Uln { get; set; }
+        public string ProviderName { get; set; }
+        public int ProviderUkprn { get; set; }
+        public string TlevelName { get; set; }
+        public int AcademicYear { get; set; }
+        public int AcademicYearTo { get; set; }
+        public string DisplayAcademicYear { get; set; }
+        public string ContactName { get; set; }       
+        public string RequestDate { get; set; }
```

Comment:
Why is this RequestDate a string? It should be a DateTime.
If it is because it is typed in the page it should be already validated and parsed by the time we call the internal API